### PR TITLE
PPO notebook added

### PR DESCRIPTION
**Pull request by theDebugger811**
Commit comprises:
1. PPO Notebook added
2. Gym Notebook drivergym figure added
3. Minor hyperparam updates

### CODE REVIEW

File: examples/RL/ppo_policy_training.ipynb [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+    "train_sim_cfg = SimulationConfigGym()\n",
+    "train_sim_cfg.num_simulation_steps = train_eps_length + 1\n",
+    "env_kwargs = {'env_config_path': env_config_path, 'use_kinematic': True, 'sim_cfg': train_sim_cfg}\n",
+    "env = make_vec_env(\"L5-CLE-v0\", env_kwargs=env_kwargs, n_envs=train_envs,\n",
+    "                   vec_env_cls=SubprocVecEnv, vec_env_kwargs={\"start_method\": \"fork\"})\n",
+    "\n",
+    "# make eval env\n",
+    "validation_sim_cfg = SimulationConfigGym()\n",
+    "validation_sim_cfg.num_simulation_steps = None\n",
+    "eval_env_kwargs = {'env_config_path': env_config_path, 'use_kinematic': True,  'return_info': True, 'train': False, 'sim_cfg': validation_sim_cfg}\n",
```
Comment (perone) [inline examples/RL/ppo_policy_training.ipynb:113]: Long line here, consider breaking it

File: examples/RL/ppo_policy_training.ipynb [hunk 2]
```diff
@@ -0,0 +1,387 @@
...
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "callback_list = []\n",
+    "\n",
+    "# Save Model Periodically\n",
+    "save_freq = 100000\n",
+    "save_path = './logs/'\n",
+    "output = 'PPO'\n",
+    "checkpoint_callback = CheckpointCallback(save_freq=(save_freq // train_envs), save_path=save_path, name_prefix=output)\n",
```
Comment (perone) [inline examples/RL/ppo_policy_training.ipynb:246]: long line

File: examples/RL/ppo_policy_training.ipynb [hunk 3]
```diff
@@ -0,0 +1,387 @@
...
+    "callback_list.append(checkpoint_callback)\n",
+    "\n",
+    "# Eval Model Periodically\n",
+    "eval_freq = 100000\n",
+    "n_eval_episodes = 1\n",
+    "val_eval_callback = L5KitEvalCallback(eval_env, eval_freq=(eval_freq // train_envs), n_eval_episodes=n_eval_episodes, n_eval_envs=eval_envs)\n",
```
Comment (perone) [inline examples/RL/ppo_policy_training.ipynb:252]: long line

File: examples/RL/ppo_policy_training.ipynb [hunk 4]
```diff
@@ -0,0 +1,387 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fcd2b424",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rollout_sim_cfg = SimulationConfigGym()\n",
+    "rollout_sim_cfg.num_simulation_steps = None\n",
+    "rollout_env = gym.make(\"L5-CLE-v0\", env_config_path=env_config_path, sim_cfg=rollout_sim_cfg, use_kinematic=True, train=False, return_info=True)\n",
```
Comment (perone) [inline examples/RL/ppo_policy_training.ipynb:315]: long line

File: examples/RL/ppo_policy_training.ipynb [hunk 5]
```diff
@@ -0,0 +1,387 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7e383cff",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# might change with different rasterizer\n",
+    "mapAPI = rollout_env.dataset.rasterizer.sem_rast.mapAPI\n",
```
Comment (perone) [inline examples/RL/ppo_policy_training.ipynb:354]: use `map_API` or `map_api`, as camel case is not that common in Pythonn

File: examples/RL/ppo_policy_training.ipynb [hunk 6]
```diff
@@ -0,0 +1,387 @@
+{
```
Comment (perone) [inline examples/RL/ppo_policy_training.ipynb:1]: It got a little confusing to have a `ppo.py` and a `ppo_policy_training.ipynb` here in the same folder. I would:

- Rename `ppo.py` to another more meaningful name, what is this file doing ? Just looking at it, it looks like it is a PPO implementation, but it isn't;
- create a new folder here called "notebooks" and move this notebook to inside of it and adjust the paths

**Comment by theDebugger811** [comment]
Thanks for the quick review @perone, changes have been incorporated.

File: examples/RL/ppo_policy_training.ipynb (hunk 6 above)
Comment (theDebugger811) [reply examples/RL/ppo_policy_training.ipynb]: - `ppo.py` updated to `ppo_training.py`
- new folder `notebooks` created

**Review by perone — APPROVED** [review]

**Pull request merged**